osquery/remote/http/http_client.cpp

```diff
@@ -47,6 +47,9 @@ void Client::postResponseHandler(boost_system::error_code const& ec) {
       (ec.value() == kSSLShortReadError)) {
     // Ignoring short read error, set ec_ to success.
     ec_.clear();
+    // close connection for security reason.
+    LOG(INFO) << "SSL SHORT_READ_ERROR: http_client closing socket";
```

Comment:
A more descriptive error and what the user should do in this case would be helpful. If no user action is necessary, it shouldn't be `INFO`.